### PR TITLE
cPanel Basics

### PR DESCRIPTION
**Pull request by danHavercroft**
- Added up-to-date guide on connecting to WHM/cPanel
 - Added guide on the inital setup page
 - Added page on creating cPanel accounts
 - Added page on creating FTP accounts

### CODE REVIEW

File: source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Creating An FTP Account In cPanel
+
+You might need to create FTP accounts in cPanel, so that you and/or your developers can upload and download content from your website.  
+
+In order to do this, start by logging into the cPanel account that you want to create the FTP account for. [You can find our guide on this here!](cpanel_connect)
```
Comment (CountBacula) [inline source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md]: link needs updating to relative/path/file.html

File: source/operatingsystems/linux/controlpanels/cpanel_initial_setup.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# WHM First Time Setup
+
+When you first login to WHM, you'll be presented with a page, asking for an email address, and 2 nameservers.
```
Comment (CountBacula) [inline source/operatingsystems/linux/controlpanels/cpanel_initial_setup.md]: login = log in

File: source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+You will be looking at this page:
+
+![FTP Accpunt Page](files/ftp_accounts_page.JPG)
+
+Firstly, choose and input a username and password for your account.
+
+The next box is for "Quota". This determines how much bandwidth the FTP account can use. By default, this can be left as unlimited.
+
+The last box is "Directory". Unless the account you're adding should only be allowed to access a specific subdirectory of the site, this can be left as blank. This will allow the FTP user to access all the files in the site's home directory.
```
Comment (CountBacula) [inline source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md]: These steps should likely be swapped to match the order on the screenshot.

**Review by CountBacula — CHANGES_REQUESTED** [review]
Few minor changes needed but otherwise looking good!

File: source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md (hunk 1 above)
Comment (danHavercroft) [reply source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md]: Resolved in 8a8ab76

File: source/operatingsystems/linux/controlpanels/cpanel_initial_setup.md (hunk 2 above)
Comment (danHavercroft) [reply source/operatingsystems/linux/controlpanels/cpanel_initial_setup.md]: Resolved in 200ac58

File: source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md (hunk 3 above)
Comment (danHavercroft) [reply source/operatingsystems/linux/controlpanels/cpanel_ftp_account.md]: Resolved in 2216c79

**Review by CountBacula — APPROVED** [review]

**Review by CountBacula — APPROVED** [review]

**Pull request merged**